### DISCUSSION TITLE
Is it possible to blacklist binaries from showing in search?

### DISCUSSION THREAD

**Discussion by Cecron** [Ulauncher/Ulauncher#1121] [Q&A] [answered]
Is it possible to remove specific application entries from the search list?

I have Flameshot installed, but this tool needs to be started as "flameshot gui" so I created a shortcut (with keyword flame) for it which works great.
But when typing "flame" the original (non-working) entry also shows up. Is it possible to remove this non-working entry?

Coming from executor.dk on Windows I was thrilled to find ulauncher!
Thanks!

**Comment by friday** [comment, marked as answer]
It's the same for all launchers. You can hide it for all of them: https://ostechnix.com/how-to-hide-applications-from-ubuntu-dash/

What you probably should be doing is to change the Exec line instead of adding Hidden=True. Then you won't need the shortcut and it'll work for all launchers.

**Reply by Cecron** [reply]
That worked great!
Being new to the Ubuntu desktop environment, I really appreciate your detailed answer!

Thanks again.
